VSocks.md

```diff
@@ -0,0 +1,133 @@
+# Kata Containers and vsocks
+
+- [Introduction](#introduction)
+    - [proxy communication diagram](#proxy-communication-diagram)
+    - [vsock communication diagram](#vsock-communication-diagram)
+- [System requirements](#system-requirements)
+- [Advantages of using vsocks](#advantages-of-using-vsocks)
+    - [High density](#high-density)
+    - [Reliability](#reliability)
+
+## Introduction
+
+There are two different ways processes in the virtual machine can communicate
+with processes in the host. The first one is by using serial ports, where the
+processes in the virtual machine can read/write data from/to a serial port device
+and the processes in the host can read/write data from/to a unix socket. This
+solution has been part of QEMU from long time ago and almost all GNU/Linux
+distributions have support for serial ports, making it the most portable solution.
+However, there is a limitation that impact its usage and implementation. The
+communication between processes is serial, making impossible that two or more
```

Comment:
Lines 19, 20, and 21 - "However, the serial link limits read/write access to one process at a time."